### PR TITLE
refactor: updated readmes to include download of specific examples

### PR DESCRIPTION
**Pull request by ruheni**
Downloading specific example is now possible via `curl`  🚀🚀

### CODE REVIEW

**Comment by nikolasburk** [comment]
Thanks Alex, you rock 💪

File: .github/readmes/javascript/rest-express/README.md [hunk 1]
```diff
@@ -2,9 +2,13 @@
 
 This example shows how to implement a **REST API** using [Express](https://expressjs.com/) and [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client). It uses a SQLite database file with some initial dummy data which you can find at [`./prisma/dev.db`](./prisma/dev.db).
 
+__INLINE(../_setup-0.md)__
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/javascript/rest-express`
```
Comment (janpio) [inline .github/readmes/javascript/rest-express/README.md:6]: The ` at the end is probably wrong here?

File: .github/readmes/typescript/graphql-hapi/README.md [hunk 2]
```diff
@@ -2,13 +2,17 @@
 
 This example shows how to implement an **GraphQL Server Example with Hapi** based on [hapi](https://hapi.dev/), [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client), [Apollo Server Hapi](https://github.com/apollographql/apollo-server/tree/main/packages/apollo-server-hapi) and [Nexus Schema](https://nexusjs.org/docs/). It is based on a SQLite database, you can find the database file with some dummy data at [`./prisma/dev.db`](./prisma/dev.db).
 
+__INLINE(../_setup-0.md)__
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/typescript/graphql-hapi
 __INLINE(../_setup-1.md)__
-cd prisma-examples/typescript/graphql-hapi
+d graphql-hapi
```
Comment (janpio) [inline .github/readmes/typescript/graphql-hapi/README.md:8]: missing `c` here

File: .github/readmes/javascript/rest-express/README.md (hunk 1 above)
Comment (ruheni) [reply .github/readmes/javascript/rest-express/README.md:6]: Thanks for catching that. I must have missed that

File: .github/readmes/typescript/graphql-hapi/README.md (hunk 2 above)
Comment (ruheni) [reply .github/readmes/typescript/graphql-hapi/README.md:8]: Thank you for catching that 🙃

**Pull request merged**